source/puppet/4.1/reference/lang_facts_and_builtin_vars.markdown

```diff
@@ -79,11 +78,11 @@ All facts appear in Puppet as [top-scope variables][topscope]. They can be acces
 
 Example, with the osfamily fact:
 
-~~~ ruby
-    if $osfamily == 'redhat' {
-      # ...
-    }
-~~~
+``` puppet
+if $osfamily == 'redhat' {
```

Comment:
The context here is to explicitly refer to this as a "classic `$fact_name` fact" distinct from the following section, which covers "the `$facts['fact_name']` hash". I might be missing something here, but I'm not sure that replacing this specific $fact_name example with a $facts['fact_name'] example makes sense.